_commands/tools/units.md

```diff
@@ -3,22 +3,37 @@
 
 units
 ---
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`units` converts quantities from one unit to another unit. It can be used interactively or non-interactively.
+
 
-<!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ units '10 meters' 'feet'
+# displays * 32.808399 
+           / .03048
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+To view available units, look in /usr/share/units/definitions.units
+
+### Using `units` Interactively
+Brings up an interactive prompt. The line with the * gives the result. The line with the / gives the inverse of the conversion factor. In the example below, 10 meters in 32.808399 feet, and 1 foot is .03038 decameters.
```

Comment:
could you describe the interactive prompt more precisely? i.e. that you'd need to supply information in the order of what you have, *then* what you want, and you will continue to be reprompted until you exit